cmd/lncli/commands.go

```diff
@@ -18,6 +18,8 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/lightningnetwork/lnd/lnrpc/routerrpc"
```

Comment:
is this commit supposed to be included, or removed?

Reply:
Removed, eventually. I keep it here for easy testing (possibly by reviewers :stuck_out_tongue: )

Reply:
Removed now